BookEcommerce.Models/DTOs/CreateAccountDTO.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookEcommerce.Models.DTOs
+{
+    internal class CreateAccountDTO
```

Comment:
class name should not an action

Reply:
this class is redundant so i've removed it.

Reply:
btw i will change all class names if it went as the same name like this